akka-projection-cluster/src/main/scala/akka/projection/cluster/scaladsl/ClusterProjectionRunner.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.cluster.scaladsl
+
+import scala.collection.immutable
+
+import akka.actor.typed.ActorSystem
+import akka.cluster.sharding.typed.ShardedDaemonProcessSettings
+import akka.projection.Projection
+import akka.projection.cluster.internal.ClusterProjectionRunnerImpl
+
+object ClusterProjectionRunner {
+
+  def init[Envelope](
+      system: ActorSystem[_],
+      projectionName: String,
+      projections: immutable.IndexedSeq[() => Projection[Envelope]]): Unit =
+    init(system, projectionName, projections, ShardedDaemonProcessSettings(system))
```

Comment:
This is minimal now, but will grow in complexity once we have the API to start and stop. I prefer to keep it as long we are in ApiMayChange phase.

Reply:
but we don't know how that will look like so we could do the opposite, start without it and then add it if it's needed.

The start/pause/stop functionality will probably not be specific to projections be can be for all ShardedDaemonProcess

Reply:
I changed my mind about it because of discussion [here](https://github.com/akka/akka-projection/pull/99#discussion_r422119527)

If we make it `Int => Projection`, than let's make it `Int => Behavior` and just use `ShardedDaemonProcess`. 

Then, the stop/start API will probably added such a thing as a runner and solve all the hard parts. 

The user can then choose, `ShardedDaemonProcess` directly and no control over the projeciton or `ClusterProjectionRunner` and control over it. 

Or we add control to `ShardedDaemonProcess`

Reply:
I will open another PR with only the ProjectionBehavior and keep this one on the side.

Reply:
That was my thread of thought as, well, starting from Seq => Int => Projection => Behavior

we still want some integration test here that is using ShardedDaemonProcess